subprojects/core/src/main/java/org/gradle/api/internal/FeaturePreviews.java

```diff
@@ -28,7 +28,7 @@
     public enum Feature {
         GRADLE_METADATA(false),
         GROOVY_COMPILATION_AVOIDANCE(true),
-        ONE_LOCKFILE_PER_PROJECT(true),
+        ONE_LOCKFILE_PER_PROJECT(false),
```

Comment:
Is this now still needed?

Reply:
Yes, it now prints a deprecation message when used and we remove it in 8.0. Like `GRADLE_METADATA` which has to be removed for 7.0.